google/cloud/spanner_dbapi/connection.py

```diff
@@ -349,6 +349,28 @@ def run_statement(self, statement, retried=False):
             ResultsChecksum() if retried else statement.checksum,
         )
 
+    def validate(self):
+        """
+        Execute a minimal request to check if the connection
+        is valid and the related database is reachable.
+
+        Raise an exception in case if the connection is closed,
+        invalid, target database is not found, or the request result
+        is incorrect.
+
+        :raises: :class:`InterfaceError`: if this connection is closed.
+        :raises: :class:`OperationalError`: if the request result is incorrect.
```

Comment:
Please also include `NotFound` if instance or database are non-existent. The previous paragraph mentions that an exception is raised in this case so it's good to explicitly state what this exception will be.

Reply:
Done